src/scrapers/base-scraper-with-browser.ts

```diff
@@ -169,6 +131,57 @@ class BaseScraperWithBrowser<TCredentials extends ScraperCredentials> extends Ba
     });
   }
 
+  private async initializePage() {
+    debug('initialize browser page');
+    if ('browserContext' in this.options) {
+      debug('Using the browser context provided in options');
+      return this.options.browserContext.newPage();
+    }
+
+    if ('browser' in this.options) {
+      debug('Using the browser instance provided in options');
+      const { browser } = this.options;
```

Comment:
same here

Reply:
Same as in https://github.com/eshaham/israeli-bank-scrapers/pull/884#discussion_r1921944357 - but with `ExternalBrowserOptions`